src/test/java/com/wikia/webdriver/elements/oasis/pages/TemplateEditPage.java

```diff
@@ -3,23 +3,30 @@
 import com.wikia.webdriver.common.contentpatterns.URLsContent;
 import com.wikia.webdriver.elements.oasis.components.templateclassificiation.TemplateClassification;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
-
+import org.openqa.selenium.By;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
+import java.util.List;
+
 public class TemplateEditPage extends WikiBasePageObject {
 
   @FindBy(css = "#editarea")
   private WebElement editArea;
 
+  @FindBy(css = "#PermissionsError")
+  private WebElement permissionsErrorNotice;
+
   public TemplateEditPage() {
     super();
   }
 
   public TemplateEditPage open(String templateName) {
     getUrl(urlBuilder.appendQueryStringToURL(String.format("%s%s%s:%s", urlBuilder.getUrlForWiki(),
-        URLsContent.WIKI_DIR, URLsContent.TEMPLATE_NAMESPACE, templateName),
-        URLsContent.ACTION_EDIT));
+                                                           URLsContent.WIKI_DIR,
```

Comment:
Seems like your formatted is a bit off, sure you are using the default wikia one?